.github/workflows/versionincrement.yaml

```diff
@@ -0,0 +1,78 @@
+---
+name: Version Increment
+
+on:
+  workflow_dispatch:
+    inputs:
+      logLevel:
+        description: Log level
+        required: true
+        default: warning
+        type: choice
+        options:
+          - info
+          - warning
+          - debug
+
+jobs:
+  plugin-version-increment-sync:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        repo: [common-utils, job-scheduler, ml-commons, performance-analyzer, performance-analyzer-rca, security, alerting, index-management, sql, observability, asynchronous-search, k-NN, anomaly-detection, cross-cluster-replication, dashboards-reports, geospatial, notifications]
```

Comment:
Can we put each item on this list on a new line with `-`? Will be easier to add/remove. Sort them in alphabetical order too :)

Reply:
Sure @dblock thats doable.